easybuild/easyconfigs/l/LAMMPS/LAMMPS-2Aug2023-foss-2023a-kokkos.eb

```diff
@@ -0,0 +1,167 @@
+name = 'LAMMPS'
+version = '2Aug2023'
```

Comment:
```suggestion
version = '2Aug2023_update2'
```

Reply:
Note that we'll need to update `translate_lammps_version` in the LAMMPS easyblock accordingly to strip off the `_updateX` from the version (or translate it to `.X`)

Reply:
updated the easyblock in pr https://github.com/easybuilders/easybuild-easyblocks/pull/3036/files to accomodate add _updateX in version